octoprint_mrbeam/static/js/lib/compare-versions.js

```diff
@@ -0,0 +1,165 @@
+/* global define */
+(function (root, factory) {
```

Comment:
is this code extracted from some public library? if so I think it would be good to write a comment pointing to the maintainer of the code and the repo...
If not, it would be nice to add some tests as we now know 😄

Reply:
This is an npm package that also has browser support. I added all the details in the file.